wallets/metamask/cypress.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from 'cypress'
+import installSynpress from './src/cypress/installSynpress'
+
+export default defineConfig({
+  chromeWebSecurity: false,
+  e2e: {
+    baseUrl: 'http://localhost:9999',
+    specPattern: 'test/cypress/**/*.cy.{js,jsx,ts,tsx}',
+    supportFile: 'src/cypress/support/e2e.{js,jsx,ts,tsx}',
+    testIsolation: false,
+    async setupNodeEvents(on, config) {
+      return installSynpress(on, config)
```

Comment:
is `installSynpress` a good name here?
maybe `configureSynpress` would be better?

Reply:
what do you think about "configureBeforeSynpress"?